docs/we_made_this.md

```diff
@@ -47,7 +47,7 @@ Discover how easy it is to quickly “power-up” your Node.js Lambda functions
 
 > **Author: [Eoin Shanaghy](https://twitter.com/eoins){target="_blank"} :material-twitter:**
 
-This post shows how to use AWS Lambda Powertools to quickly build Well-Architected serverless applications.
+This post shows how to use AWS Lambda Powertools to quickly build Well-Architected Serverless applications.
```

Comment:
```suggestion
This post shows how to use Powertools for AWS Lambda to quickly build Well-Architected Serverless applications.
```

I know this came from the customer, but in Python we normalize titles on our side. Even though this remains the old way on the customer side